Added Tracing setup instructions to Redis (#5794)
DataDog/integrations-core: redisdb/README.md

| -------------- | --------------------------------------------------- |
| `<LOG_CONFIG>` | `{"source": "redis", "service": "<YOUR_APP_NAME>"}` |

##### Trace collection

> APM for containerized apps is supported on hosts running Agent 6 and above but will need some extra configuration to begin collecting traces.

Reviewer: ```suggestion
APM for containerized apps is supported on hosts running Agent v6+ but requires extra configuration to begin collecting traces.
```

> Required environment variables on the agent container:

Reviewer: ```suggestion
Required environment variables on the Agent container:
```

| Parameter            | Value                                                                      |
| -------------------- | -------------------------------------------------------------------------- |
| `<DD_API_KEY>` | `api_key`                                                                  |
| `<DD_APM_ENABLED>`      | true                                                              |
| `<DD_APM_NON_LOCAL_TRAFFIC>`  | true |

See [Tracing Docker Applications][14] and the [Kubernetes Daemon Setup][15] for a complete list of available environment variables and configuration.

> Then, [instrument your application container][7] and set the DD_AGENT_HOST environment variable to the name of your agent container. 

Reviewer: ```suggestion
Then, [instrument your application container][7] and set `DD_AGENT_HOST` to the name of your Agent container.
```
